pkg/relayer/session/session.go

```diff
@@ -157,3 +191,42 @@ func (rp *relayerSessionsManager) validateConfig() error {
 
 	return nil
 }
+
+// waitForBlock blocks until the block at the given height (or greater) is
+// observed as having been committed.
+func (rs *relayerSessionsManager) waitForBlock(ctx context.Context, height int64) client.Block {
```

Comment:
I almost wanted to create a suggestion to rename this to `blockForBlock` :)

Reply:
:joy: